### ISSUE TITLE
Fix Body encoding issue for exported json files in Blob Storage

### ISSUE THREAD

**Issue by jomit** [jomit/smart-factory-accelerator#1]
The Body property in the json files exported using Continuous Blob data export from IoT Central is utf-8 encoded.

Need to add the contentType and contentEncoding properties to all outbound messages as shown [here](https://docs.microsoft.com/en-us/azure/iot-central/core/howto-export-data#telemetry) to fix it.

**Comment by dhanow** [comment]
I debugged this, (there was a bug in the code which I fixed) even after explicitly setting the ContentType to application/json and ContentEncoding to UTF-8, I see that when you write to blob storage the body is base64 encrypted. The same behavior is observed with IoT-Hub. I believe this may be the default behavior. When I read the messages from ASA the messages get decrypted automatically and show up properly in the query output. Below are the screenshots.

![blob-output](https://user-images.githubusercontent.com/28070070/87993520-55289800-cab0-11ea-87b2-17ccc451a3cd.png)

![asa-output](https://user-images.githubusercontent.com/28070070/87993560-74272a00-cab0-11ea-9779-e02267a5eada.png)
